.github/CODEOWNERS

```
@@ -1,5 +1,62 @@
-/.github/ @elastic/doc-leads
-/.github/workflows/ @elastic/docs-engineering
+# Unless a later match takes precedence, someone on the docs team must review all PRs to this repo
+*                                           @elastic/docs
 
-/serverless/ @elastic/platform-docs
-/serverless/serverless-changelog.asciidoc @elastic/docs
+# Templates, workflows, and config
+/.github/                                   @elastic/doc-leads
+/.github/workflows/                         @elastic/docs-engineering
+/acrolinx/                                  @elastic/doc-leads
+
+# The docs
+/cloud-account/                             @elastic/admin-docs
+/deploy-manage/                             @elastic/admin-docs
+/explore-analyze/                           @elastic/core-docs @elastic/experience-docs 
+/extend/                                    @elastic/core-docs
+/get-started/                               @elastic/core-docs
+
+/manage-data/                               @elastic/admin-docs
```

concepts like mapping and text analysis are owned by the search engine team.

```suggestion
/manage-data/                               @elastic/admin-docs
/manage-data/data-store/            @elastic/admin-docs @elastic/developer-docs 
```